src/main/kotlin/no/nav/syfo/sykmelding/UpdateFnrService.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.syfo.sykmelding
+
+import no.nav.syfo.db.DatabaseInterfacePostgres
+import no.nav.syfo.pdl.service.PdlPersonService
+import no.nav.syfo.sykmelding.db.updateFnr
+import org.slf4j.LoggerFactory
+
+class UpdateFnrService(
+    private val pdlPersonService: PdlPersonService,
+    private val syfoSmRegisterDb: DatabaseInterfacePostgres
+) {
+
+    private val log = LoggerFactory.getLogger(UpdateFnrService::class.java)
+
+    suspend fun updateFnr(accessToken: String, fnr: String, nyttFnr: String): Boolean {
+
+        val pdlPerson = pdlPersonService.getPdlPerson(nyttFnr, accessToken)
+
+        if (pdlPerson.fnr != nyttFnr) {
```

Comment:
Hva er det vi egentlig sjekker her..? Ser ut som at vi bare sjekker at nytt fnr finnes i PDL? Hadde det ikke vært lurt å heller sjekke at gammelt fnr er et historisk fnr for det nye fnr-et..?

Reply:
Sorry, jeg burde slengt på en kommentar! Ja, vi sjekker at det nye fødselsnummeret du vil skifte til finnes i PDL. 

I de tilfellene hvor jeg har skiftet FNR for folk manuelt har ikke det gamle fødselsnummeret ligget som historisk i PDL, så jeg frykter at det kanskje vil skape unødvendig mye ekstraarbeid (dvs. at vi ikke kan bruke API-et) hvis vi krever at det gamle også finnes PDL.